.github/workflows/main.yml

```diff
@@ -0,0 +1,16 @@
+name: Build packages
+on:
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
```

Comment:
This has no contents - what effect does it have?

Reply:
It allows to run the workflow manually. I removed it with the latest commit.

Reply:
Running the workflow manually is good IMO - could this go back in please?

Reply:
I added it in the latest commit.